pulseapi/entries/views.py

```diff
@@ -182,7 +182,7 @@ def get_queryset(self):
         ids = self.request.query_params.get('ids', None)
         if ids is not None:
             ids = [ int(x) for x in ids.split(',') ]
-            queryset = Entry.objects.filter(pk__in=ids)
+            queryset = Entry.objects.filter(pk__in=ids, is_approved=True)
```

Comment:
wonder if we should make this conditional on staff permissions, but maybe that should be followup